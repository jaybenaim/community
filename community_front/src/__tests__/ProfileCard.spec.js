import React from "react";
import { create } from "react-test-renderer";

describe("ProfileCard Component", () => {
  test("ProfileCard has a button that takes users to the profile page", () => {
    const button = create(
      <button variant="primary" onClick={[Function]}>
        Click to see more
      </button>
    );
    expect(button.toJSON()).toMatchSnapshot();
  });
});
