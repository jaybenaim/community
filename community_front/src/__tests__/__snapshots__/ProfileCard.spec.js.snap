// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileCard Component ProfileCard has a button that takes users to the profile page 1`] = `
<button
  onClick={
    Array [
      [Function],
    ]
  }
  variant="primary"
>
  Click to see more
</button>
`;
